doc/source/getting_started/installation.rst

```diff
@@ -0,0 +1,101 @@
+.. _installation:
+
+============
+Installation
+============
+
+The ``ansys-fluent-core`` package supports Python 3.7 through
+Python 3.10 on Windows and Linux.
+
+.. note::
+   This page provides information for quickly installing and launching
+   the ``ansys-fluent-core`` package. Additional PyFluent packages, which
+   are described later on this page, can also be installed. For
+   information on installing all PyFluent packages in a virtual environment,
+   see :ref:`faqs_install` in :ref:`faqs`.
+
+Install the package
+-------------------
+Install the latest ``ansys-fluent-core`` package from
+`PyPi <https://pypi.org/project/ansys-fluent-core/>`_ with this code:
+
+.. code::
+
+   pip install ansys-fluent-core
+
+If you plan on doing local *development* of PyFluent with Git, install the
+latest ``ansys-fluent-core`` package with this code:
+
+.. code:: console
+
+   git clone https://github.com/pyansys/pyfluent.git
+   cd pyfluent
+   pip install pip -U
+   pip install -e .
+   python codegen/allapigen.py  # Generates the API files
+
+
+Any changes that you make locally are reflected in your setup after you restart
+the Python kernel.
+
+Launch Fluent
+-------------
+To launch Fluent from PyFluent, use the :func:`launch_fluent() <ansys.fluent.core.launcher.launcher.launch_fluent>`
+method:
+
+.. code:: python
+
+  import ansys.fluent.core as pyfluent
+  solver = pyfluent.launch_fluent(precision="double", processor_count=2, mode="solver")
+  solver.health_check_service.is_serving
+
+To locate the Fluent installation, PyFluent automatically uses the ``AWP_ROOT<ver>``
+environment variable, where ``<ver>`` is the three-digit format for the release.
+For example, ``AWP_ROOT222`` is the environment variable for the 2022 R2 release. 
+
+On a Windows system, this environment variable is configured when a release is installed.
+
+On a Linux system, you must configure this environment variable to point to the absolute
+path of the installed release. For example, for the 2022 R2 release, you would likely set
+the ``AWP_ROOT222`` environment variable to point to ``C:\Program Files\ANSYS Inc\v222``.
+
+To use a non-default installation location, either set the ``AWP_ROOT<ver>`` environment variable
```

Comment:
I think we should take this paragraph out.  It was added for the use case where Fluent developers have a local Fluent build area.